src/analysis/normed_space/star/multiplier.lean

```diff
@@ -0,0 +1,414 @@
+/-
+Copyright (c) 2022 Jireh Loreaux. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jireh Loreaux, Jon Bannon
+-/
+
+import algebra.star.star_alg_hom
+import analysis.normed_space.star.basic
+import analysis.normed_space.operator_norm
+import analysis.special_functions.pow
+
+/-!
+# Multiplier Algebra of a C⋆-algebra
+
+Define the multiplier algebra of a C⋆-algebra as the algebra (over `𝕜`) of double centralizers,
+for which we provide the localized notation `𝓜(𝕜, A)`.  A double centralizer is a pair of
+continuous linear maps `L R : A →L[𝕜] A` satisfying the intertwining condition `R x * y = x * L y`.
+
+There is a natural embedding `A → 𝓜(𝕜, A)` which sends `a : A` to the continuous linear maps
+`L R : A →L[𝕜] A` given by left and right multiplication by `a`, and we provide this map as a
+coercion.
+
+The multiplier algebra corresponds to a non-commutative Stone–Čech compactification in the sense
+that when the algebra `A` is commutative, it can be identified with `C₀(X, ℂ)` for some locally
+compact Hausdorff space `X`, and in that case `𝓜(𝕜, A)` can be identified with `C(β X, ℂ)`.
+
+## Implementation notes
+
+## TODO
+
++ define a type synonym for `𝓜(𝕜, A)` which is equipped with the strict uniform space structure
+  and show it is complete
++ show that the image of `A` in `𝓜(𝕜, A)` is an essential ideal
++ prove the universal property of `𝓜(𝕜, A)`
+* Construct a double centralizer from a pair of maps `L : A → A`, `R : A → A` satisfying the
+  centrality condition `∀ x y, R x * y = x * L y`.
```

Comment:
I'm sure you know what you mean by this, but the way I read it this TODO just refers to `double_centralizer.mk`. Do you mean that the maps `L R : A → A` are just plain functions?

Reply:
Yes, it's kind of magic that linearity and continuity come for free, at least when `A` is a C⋆-algebra. The argument is standard in C⋆-algebra theory, but it's not trivial and requires approximate units, which we don't have yet.

Reply:
Oh okay! Pretty cool :^)